README.md

```diff
@@ -32,7 +32,7 @@ If you don’t update, **permissions will default to OR behavior** in the new ve
 [![Java](https://raw.githubusercontent.com/intergrav/devins-badges/v3/assets/cozy/built-with/java8_vector.svg)](https://www.java.com/)
 
 ### LiteCommands
-Annotation based command framework for Velocity, Bukkit, Paper, BungeeCord, Minestom, Sponge, Fabric, JDA and future implementations.
+Annotation based command framework for Velocity, Bukkit, Paper, BungeeCord, Minestom, Sponge, Fabric, JDA, TelegramBots and future implementations.
```

Comment:
We should probably make it a bullet list now 😮

Reply:
There is a bullet list below already, I guess we shouldn't make a second one